src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
> Given this only triggers when an uncaught warning or exception is passed through, I expect this to have very little impact in actual consumer code. 

@clue the additional warning is emitted even when the exception **is** caught, see the example in [my previous comment](https://github.com/reactphp/event-loop/pull/65#discussion_r99604567).